engines/wintermute/base/gfx/osystem/base_surface_osystem.cpp

```diff
@@ -274,6 +273,17 @@ uint32 BaseSurfaceOSystem::getPixelAt(Graphics::Surface *surface, int x, int y)
 	return 0;
 }
 
+//////////////////////////////////////////////////////////////////////////
+bool BaseSurfaceOSystem::getPixel(int x, int y, byte *r, byte *g, byte *b, byte *a)
+{
+	uint32 pixel = getPixelAt(_surface, x, y);
+	*r = (pixel >> 24) & 255;
```

Comment:
Is this endian-safe?

Reply:
Probably not. Thanks, I'll look into it...

Reply:
Fixed at b6197ae.